### PR TITLE
Fixes #23391, #23251 - adjust breadcrumbs switcher to pf design

### PR DESCRIPTION
**Pull request by amirfefer**
before:
pager_2

after:
pager_1

### CODE REVIEW

**Comment by ohadlevy** [comment]
does it make sense to fix all issues at http://projects.theforeman.org/issues/23251 as well?

**Comment by tstrachota** [comment]
+1 to fix it together with 23251. These issues are highly coupled.

@amirfefer do you plan to work on the other issue too? Could you fix them together?

**Comment by amirfefer** [comment]
I use fixed per_page value (10 resouces), and therefore the max-height is 360px with no scroll-bar.
 
fixed_switcher

**Comment by ohadlevy** [comment]
does it make sense (here on another pr) to add it to storybook? so its easily to test out it looks ?

**Comment by amirfefer** [comment]
I've added `BreadcrumbsBar` to the storybook:
https://rawgit.com/amirfefer/foreman/breadcrumbs/index.html

**Comment by amirfefer** [comment]
[test foreman]

**Comment by ohadlevy** [comment]
@rohoover can you review the storybook please?

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js
Comment (sharvit) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js]: Need to add `prop-types` and `default-props` as well

File: webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js
Comment (sharvit) [inline webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js]: 1. You don't actually run the `handleOpen` method!
    When clicking the toggle button in the storybook, I don't see an action-logger about it.

2. Do we actually need the extra prop `handleOpen`?
    `handleOpen` should be called after we already decieded to open the switcher but need the data inside it.
    Maybe it is better to use the `toggleSwitcher`, `closeSwitcher` and `loadSwitcherResourcesByResource` to demo this component?

**Review by sharvit — CHANGES_REQUESTED** [review]
Great addition @amirfefer :+1:

**Comment by amirfefer** [comment]
@sharvit - I removed `handleOpen` prop

**Comment by sharvit** [comment]
@amirfefer would it make it easier to understand if you add actions to `toggleSwitcher` and `closeSwitcher`?

**Review by Rohoover — COMMENTED** [review]
Can we use the a more square button size, as found here for Default Size?

https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/pagination.html

**Comment by amirfefer** [comment]
@Rohoover Thanks, I'll add a caption-less (no `next`/`previous` text) prop to the Pager component, with more square buttons.

**Comment by ohadlevy** [comment]
@amirfefer can you rebase please?

**Comment by ohadlevy** [comment]
@amirfefer whats the status of this PR? IMHO it should be included in #5662 as well.

**Comment by ohadlevy** [comment]
@amirfefer ping? lets try finishing all breadcrumbs remaining issues?

**Comment by amirfefer** [comment]
@ohadlevy  - i'm on it :)

**Comment by amirfefer** [comment]
@Rohoover 
The pagination buttons look more squarish:
swticher-desgin

**Comment by amirfefer** [comment]
[test foreman]

**Comment by waldenraines** [comment]
@Rohoover any other thoughts on the current implementation?  This PR is blocked on UX approval.

**Comment by amirfefer** [comment]
@ohadlevy could we merge this?
test failure seems unrelated

**Comment by ohadlevy** [comment]
@amirfefer once we get a UX ack from @theforeman/ui-ux

**Comment by ekohl** [comment]
I'd also appreciate if both typos in the commit message were fixed.

**Comment by amirfefer** [comment]
We've got :+1: from @Rohoover 
looks like an ACK for me :)

**Review by sharvit — APPROVED** [review]
LGTM! Thanks @amirfefer

File: package.json
Comment (ohadlevy) [inline package.json:9]: just noticed there is package dep, @ekohl ack? (even if this package is only used to generate the storybook and never used in building the app).

**Review by Rohoover — COMMENTED** [review]
Sorry I was on PTO. 

@amirfefer is correct, I gave a thumbs up!

LGTM

**Review by ekohl — APPROVED** [review]

**Comment by ohadlevy** [comment]
test failure not relevant, thanks @amirfefer

**Comment by ares** [comment]
Setting redmine release to 1.19, please don't forget to set the desired version after merging, thanks.

**Comment by amirfefer** [comment]
@ares Do we want to CP this into 1.18? as @ohadlevy mentioned before
basically it's a bug fix.

**Comment by ares** [comment]
Looks good and safe, but seems pretty minor given we're building RC2 already. @xprazak2 what's your take on this?

**Comment by ohadlevy** [comment]
IMHO it should be cherrypicked into 1.8-stable, I care a bit less if the
version is 1.18.0 or .1, having said all of that, this seems like a very
low risk pr to merge regardless.

**Pull request merged**
